src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -237,6 +237,9 @@ export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElemen
   ]);
 };
 
-export const displayParticipantCount = (count: number): string => {
+export const displayParticipantCount = (count: number | undefined): string => {
+  if (count === undefined) {
```

Comment:
Do we need to support `undefined`? I think allowing `undefined` was a temporary state and at this point we will always have a `count` value for a displayed concept.

At one point I had thought that we'd show all concepts and then ajax in the per-concept counts after displaying the page. But since the views are count sorted and paginated, it turns out we have to generate the counts in the API. It's possible the types haven't been updated to reflect this yet.

Reply:
We need to support `undefined` in the instances of `ConceptSearch`where we are iterating through a list of concepts. Since `count` is optional for a `SnapshotBuilderConcept` object. We can't use the previous method because `number | undefined` was going to be passed in.